library/ecp_curves.c

```diff
@@ -21,7 +21,7 @@
 #include "ecp_invasive.h"
 
 #if defined(MBEDTLS_ECP_WITH_MPI_STRUCT)
```

Comment:
Any specific reason that this has a double dependency? The code block needs both MBEDTLS_ECP_WITH_MPI_STRUCT and MBEDTLS_ECP_LIGHT to be asserted.

Are we testing possible configuration combination we are generating with the

MBEDTLS_ECP_LIGHT, MBEDTLS_ECP_WITH_MPI_STRUCT, MBEDTLS_ECP_WITH_MPI_UINT, MBEDTLS_ECP_C ?

Reply:
Actually, this PR is based on the PR #7703 , and has not rebased to the latest code.  After updated, there will not be the combination of MBEDTLS_ECP_LIGHT and MBEDTLS_ECP_C .